### PR TITLE
Added method for getting public key ids from signature

### PR DESCRIPTION
**Pull request by danhab99**
Resolving issue #1325 by adding a method for getting the KeyIDs of all the issuers of a given signature

### CODE REVIEW

File: src/signature.js [hunk 1]
```diff
@@ -51,6 +51,15 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * 
+   * @returns Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The IDs of the issuers
+   */
+  getIssuerIDs() {
```
Comment (twiss) [inline src/signature.js:60]: It might be good to be consistent with `Message#getSigningKeyIDs`, and call this function the same thing.

File: src/signature.js [hunk 2]
```diff
@@ -51,6 +51,15 @@ export class Signature {
...
+    return this.packets.map(packet => packet.issuerKeyID)
```
Comment (twiss) [inline src/signature.js:61]: Technically speaking, there could be a marker packet here, as well, which wouldn't have an issuer key ID. So it might be good to still do something like `Message#getSigningKeyIDs` does, i.e. get the signature packets here explicitly.

File: package-lock.json [hunk 3]
```diff
@@ -1,172 +1,6375 @@
 {
   "name": "openpgp",
   "version": "5.0.0-3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (twiss) [inline package-lock.json:4]: Since package.json is not changed in this PR, please don't change package-lock.json, either

File: README.md [hunk 4]
```diff
@@ -525,6 +526,34 @@ Using the private key:
 })();
 ```
 
+#### Getting the IDs of the Issuers of a signature
```
Comment (twiss) [inline README.md:529]: To be honest, while I appreciate you writing some documentation for this, I'm not sure if this operation is common enough to warrant a full example in the top-level README. Once this PR is merged and included in a release, the method will automatically be included in https://openpgpjs.org/openpgpjs/Signature.html (generated from the JSDoc), I feel like that should be enough.

File: src/signature.js [hunk 5]
```diff
@@ -51,6 +51,15 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * 
+   * @returns Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The IDs of the issuers
```
Comment (twiss) [inline src/signature.js:58]: Speaking of the JSDoc, this should be something like:
```suggestion
   * Returns an array of KeyIDs of all of the issuers who created this signature
   * @returns {Array<KeyID>} The Key IDs of the signing keys
```

**Review by twiss — COMMENTED** [review]
Hey @danhab99, thanks for the PR! See some nitpicks below.

File: README.md [hunk 6]
```diff
@@ -5,24 +5,25 @@ OpenPGP.js [![BrowserStack Status](https://automate.browserstack.com/badge.svg?b
 
 **Table of Contents**
 
-- [OpenPGP.js](#openpgpjs)
+- [OpenPGP.js ![BrowserStack Status](https://automate.browserstack.com/public-build/N1l2eHFOanVBMU9wYWxJM3ZnWERnc1lidkt5UkRqa3BralV3SWVhOGpGTT0tLVljSjE4Z3dzVmdiQjl6RWgxb2c3T2c9PQ==--5864052cd523f751b6b907d547ac9c4c5f88c8a3) [![Join the chat on Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/openpgpjs/openpgpjs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)](#openpgpjs--)
```
Comment (twiss) [inline README.md:8]: Thanks! Could you revert these changes to the table of contents as well?

File: src/signature.js [hunk 7]
```diff
@@ -51,6 +51,14 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The Key IDs of the signing keys
+   */
+  getSigningKeyIDs() {
+    return this.packets.filter(packet => packet.tag !== enums.packet.marker).map(packet => packet.issuerKeyID)
```
Comment (twiss) [inline src/signature.js:60]: And then finally, I would prefer to select the signature packets explicitly, rather than the other way around. See `Message#getSigningKeyIDs` for inspiration. (Also, there's a final semicolon missing.)

Comment (danhab99) [reply src/signature.js]: I don't quite understand how `Message#getSigningKeyIDs` relates to my solution.

Comment (danhab99) [reply src/signature.js]: figured it out
